cypress/integration/07-account_creation_spec.js

```diff
@@ -78,6 +78,12 @@ describe("Okta account creation", () => {
       cy.visit("/uac/?activationToken=h971awbXda7y7jGaxN8f");
       cy.contains("Create your password");
     });
+    it("contains no accessibility violations", () => {
+      // Test a11y on the Account Creation page
+      // We only need to check a11y once for this page workflow
```

Comment:
I think we may want to check the other pages as well - IIRC, each of these `it` calls will render a new page.

Reply:
@emmastephenson added, ready for re-review!